packages/alea-frontend/pages/api/get-students-enrolled-in-course.ts

```diff
@@ -1,23 +1,67 @@
+
+import { Action, CURRENT_TERM, getResourceId, ResourceName } from '@stex-react/utils';
 import { NextApiRequest, NextApiResponse } from 'next';
-import { executeAndEndSet500OnError } from './comment-utils';
 import { getCourseEnrollmentAcl } from '../course-home/[courseId]';
+import { getCacheKey } from './acl-utils/acl-common-utils';
+import { returnAclIdForResourceIdAndActionId } from './acl-utils/resourceaccess-utils/resource-common-utils';
+
+const courseIds = ['ai-1', 'ai-2', 'lbs', 'iwgs-1', 'iwgs-2'];
+
+const instructorResourceAndAction = [
```

Comment:
Use upper case for constants
```suggestion
const INSTRUCTOR_RESOURCE_AND_ACTION = [
```